site/sfguides/src/tempo/tempo.md

```diff
@@ -0,0 +1,228 @@
+author: mando222
+id: tempo
+summary: This is a guide on getting started with Tempo on Snowflake
+categories: Getting-Started
+environments: web
+status: Published 
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Getting Started, Security, LLGM, Intrusion Detection
+
+# Getting Started with TEMPO and Snowflake (Using Sample Data)
+<!-- ------------------------ -->
```

Comment:
@mando222 Remove " (Using Sample Data)" from the title.

Reply:
fixed